README.md

```diff
@@ -449,19 +449,17 @@ stringData:
         type: sample-instance-offering, // The service offering name
     }
 ```
-###### Custom Templates
+##### Custom Formats 
 
-For additional flexibility, you can model your `Secret` resources according to your specific needs.
+For additional flexibility, you can model your `Secret` resources according to your specific needs.<br>
 To generate a custom-formatted `Secret`, use the `secretTemplate` attribute in the `ServiceBinding` spec.
-The value of the `secretTemplate` must be a Go template. 
 
-Refer to [Go Templates](https://pkg.go.dev/text/template) for more details.
+This attribute expects a Go template as its value (for more information, see [Go Templates](https://pkg.go.dev/text/template)).<br>
 
-**Note** 
-If `secretTemplate` is used, both `secretKey` and `secretRootKey` attributes are ignored when provided.
-
-The template can use the following data: 
+**Note**<br>
+Make sure that the template is in YAML format, and that its structure is of a Kubernetes `Secret`. 
 
```

Comment:
I don't think it should be in a note.

Reply:
@I065450  It is kind of important to emphasize, we can write Important.

I would still say something about which things are customizable first and then say that in spite of this flexibility we give, some things are לא ניתן להתפשר עליהם
I mean, not exactly those words, but you get what I mean.
Then we say that it has to be a YAML doc and with the Kubernetes Secret structure.